exporter/datadogexporter/utils/trace_helpers.go

```diff
@@ -132,3 +133,118 @@ func NormalizeServiceName(service string) string {
 
 	return s
 }
+
+// NormalizeTag applies some normalization to ensure the tags match the backend requirements.
+// Specifically used for env tag currently
+// port from: https://github.com/DataDog/datadog-agent/blob/c87e93a75b1fc97f0691faf78ae8eb2c280d6f55/pkg/trace/traceutil/normalize.go#L89
```

Comment:
Could you create something (a card?) so that we have a list of ported code from the Agent?

Reply:
👍 missed this, can do